### PR TITLE
Bug 1202162 - [Settings] Convert find my device switch to use gaia-switch

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1202162

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=888c9011b9798ea237b73759f013a99037b4adfa

File: apps/settings/js/findmydevice.js [hunk 1]
```diff
@@ -108,8 +110,14 @@ define('findmydevice', ['modules/settings_utils', 'shared/settings_listener'
     },
 
     _setCanDisable: function fmd_set_can_disable(value) {
-      var checkbox = document.querySelector('#findmydevice-enabled input');
-      checkbox.disabled = !value;
+      var checkbox = document.querySelector(
+        '#findmydevice-enabled gaia-switch');
+      if (value) {
```
Comment (gasolin) [inline apps/settings/js/findmydevice.js]: can we use disabled atribute directly now?

File: apps/settings/js/findmydevice.js [hunk 2]
```diff
@@ -156,14 +164,15 @@ define('findmydevice', ['modules/settings_utils', 'shared/settings_listener'
         return;
       }
 
-      var checkbox = document.querySelector('#findmydevice-enabled input');
-      checkbox.disabled = true;
+      var checkbox = document.querySelector(
+        '#findmydevice-enabled gaia-switch');
+      checkbox.setAttribute('disabled', true);
```
Comment (gasolin) [inline apps/settings/js/findmydevice.js]: same here

File: apps/settings/js/findmydevice.js [hunk 3]
```diff
@@ -156,14 +164,15 @@ define('findmydevice', ['modules/settings_utils', 'shared/settings_listener'
...
 
       if (checkbox.checked === false) {
         wakeUpFindMyDevice(IAC_API_WAKEUP_REASON_TRY_DISABLE);
       } else {
         SettingsHelper('findmydevice.enabled').set(true, function() {
-          checkbox.disabled = false;
+          checkbox.removeAttribute('disabled');
```
Comment (gasolin) [inline apps/settings/js/findmydevice.js]: same here

File: apps/settings/js/findmydevice.js (hunk 1 above)
Comment (KevinGrandon) [inline apps/settings/js/findmydevice.js]: Yup, I will update that, thanks!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=516260cfc952c090f3e4556c1448686fa4067005

**Pull request merged**